website/apps/transcripts/views.py

```diff
@@ -330,7 +331,7 @@ def get_context_data(self, phase_number='1'):
             'act': selected_act,
         }
 
-class ErrorView(TemplateView):
+class ErrorView(MemorialMixin, TemplateView):
```

Comment:
I'm not sure this one should use the `MemorialMixin`, shouldn't we 404 on URLs we don't support for any mission?